securesignals-ima-devapp/src/test/java/com/uid2/dev/ExampleUnitTest.kt

```diff
@@ -0,0 +1,17 @@
+package com.uid2.dev
+
+import org.junit.Test
+
+import org.junit.Assert.*
+
+/**
+ * Example local unit test, which will execute on the development machine (host).
+ *
+ * See [testing documentation](http://d.android.com/tools/testing).
+ */
+class ExampleUnitTest {
```

Comment:
Example Unit test added by Android Studio. It can be removed (along with dependencies).